### PR TITLE
Add NFT page, fix local build problem

### LINKED ISSUE #2776: Computed Dict missing
Starting the wiki locally from scratch does not work due to computed-dict creation not being part of the `start` command:

```
swader@DESKTOP-0OEAV37:~/repos/polkadot-wiki$ yarn polkadot:start
yarn run v1.22.5
$ yarn generateRegistryTable; docusaurus start polkadot-wiki
$ node --experimental-json-modules scripts/generateRegistryTable.mjs > docs/build-ss58-registry.md

Starting the development server...
Error: ENOENT: no such file or directory, open '/home/swader/repos/polkadot-wiki/scripts/computed-dict.json'
    at Object.openSync (fs.js:476:3)
    at Object.readFileSync (fs.js:377:35)
    at injectPlugin (/home/swader/repos/polkadot-wiki/scripts/injectPlugin.js:117:30)
    at Object.<anonymous> (/home/swader/repos/polkadot-wiki/polkadot-wiki/docusaurus.config.js:60:27)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at module.exports (/home/swader/repos/polkadot-wiki/node_modules/import-fresh/index.js:32:59)
    at loadConfig (/home/swader/repos/polkadot-wiki/node_modules/@docusaurus/core/lib/server/config.js:17:53)
    at loadSiteConfig (/home/swader/repos/polkadot-wiki/node_modules/@docusaurus/core/lib/server/index.js:33:51)
    at loadContext (/home/swader/repos/polkadot-wiki/node_modules/@docusaurus/core/lib/server/index.js:43:69)
    at load (/home/swader/repos/polkadot-wiki/node_modules/@docusaurus/core/lib/server/index.js:182:27)
    at loadSite (/home/swader/repos/polkadot-wiki/node_modules/@docusaurus/core/lib/commands/start.js:37:34)
    at start (/home/swader/repos/polkadot-wiki/node_modules/@docusaurus/core/lib/commands/start.js:44:25)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

To fix, append `yarn fetch` to each `start` command in `package.json`.

### PR DESCRIPTION
**Pull request by Swader**
Fixes #2776 by adding fetch to local running of Polkadot and Kusama, and #2455 by adding a basic NFT page and some images.

### CODE REVIEW

File: docs/learn/learn-nft.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+The downside of having to define these standards is that they are always
+instructions for how to read a spreadsheet pretending to serve information in a
+certain way, which by definition cannot be optimized. For this reason, even on a
+good day of extremely low network congestion, interactions with NFTs on any EVM
+chain will cost a few dollars, but were on average around $100 per interaction
+(transer, mint, sale) in 2021 on Ethereum.
+
+This prevents use cases that go beyond the current craze of _digital dust
+gathering NFTs_ on Ethereum - profile pictures, generative "look once and then
+put away" art, [ENS](ens) addresses, and proof of attendance badges.
```
Comment (salmad3) [inline docs/learn/learn-nft.md]: we can add a link POAPs

File: docs/learn/learn-nft.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+exist, builders are able to natively optimize for complex NFT use cases without
+tradeoffs that would make interacting with the system prohibitively inefficient
+and expensive in other environments.
+
+The following NFT solutions exist and/or are under development in the Polkadot
+ecosystem:
+
+### Unique network
+
+[Unique network](https://unique.network/), an NFT-specific parachain offering
```
Comment (salmad3) [inline docs/learn/learn-nft.md]: would take out that they are a parachain here

**Comment by Swader** [comment]
Anything you need me to change?

**Review by jakehemmerle — APPROVED** [review]
lgtm! I'll let @DannyS03  merge it has he has been taking great care of the wiki

**Review by salmad3 — COMMENTED** [review]
Many thanks @Swader

Just minor suggestions

File: docs/learn/learn-nft.md (hunk 2 above)
Comment (Swader) [reply docs/learn/learn-nft.md]: Why's that? They're launching Quartz as a crowdloan and intend to be a parachain.

File: docs/learn/learn-nft.md (hunk 1 above)
Comment (Swader) [reply docs/learn/learn-nft.md]: Done

File: docs/learn/learn-nft.md (hunk 2 above)
Comment (salmad3) [reply docs/learn/learn-nft.md]: I'd say its best practice to classify projects as a parachain when they have a parachain slot

Comment (Swader) [reply docs/learn/learn-nft.md]: Alright, makes sense.

File: docs/learn/learn-nft.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
```
Comment (salmad3) [inline docs/learn/learn-nft.md]:
```suggestion
[Unique network](https://unique.network/), an NFT-specific blockchain offering
```

**Review by salmad3 — APPROVED** [review]

**Comment by Swader** [comment]
@DannyS03 it looks like this failed to deploy

**Comment by Swader** [comment]
Looks like action access (merging from forks) is not configured 🤔 
image

**Comment by salmad3** [comment]
Right we didn't configure it

**Comment by Swader** [comment]
I think you'll have to re-merge this somehow from a local copy of the same branch or something 🤔

**Comment by Swader** [comment]
Does not seem to be triggering a valid rebuild from other pushes.

**Comment by salmad3** [comment]
Right, looks like theres a problem with general deployment too

**Comment by salmad3** [comment]
Fix in progress

**Comment by salmad3** [comment]
@Swader thanks so much! https://wiki.polkadot.network/docs/learn/learn-nft

**Pull request merged**